src/backend/tests/brilisp/castop_fptoui.sexp

```diff
@@ -0,0 +1,15 @@
+;%Y = fptoui float 1.0E+300 to i1     ; yields undefined:1
+;%Z = fptoui float 1.04E+17 to i8     ; yields undefined:1 left to implement
+
+(brilisp
+    (define ((print int32) (n int32)))
+
+    (define ((cas_t int32) (n double))
+        (set (f_n int32) (fptoui n int32))
+        (ret f_n))
+
+    (define ((main void))
+        (set (a double) (const 123.0))
+        (set (b int32) (call cas_t a))
+        (set (tmp int32) (call print b))
```

Comment:
can you cast negative number and see what happens here?

Reply:
Tested it . Its fp to unsigned integer. For the below code.
```
(set (c double) (const -123.00000))
        (set (d int32) (call cas_t c))
        (set (tmp_1 int32) (call print d))
        (ret )))
```
For every execution `fptoui` on a negative integer has been giving the results of size similar to number of digits in 484409152. The value can be both positive and negative.